documentation/api/command/v1/commands.markdown

```diff
@@ -98,15 +99,32 @@ payload of this command.
 See [deactivate node wire format v3][deactivatev3] for more information on the
 payload of this command.
 
+### "store report", version 6
+
+The version 6 store report command differs from previous versions by changing
+from a `resource_events` property to a `resources` property. The
+`resource_events` property was a merged version of `resources` and their
+associated events `events`. This new version moves the command to use a similar
+format to a raw Puppet report, with a list of `resources` each with a `events`
+property containing a list of the resource's events.
+
+The payload is expected to be a report, containing events that occurred on
+Puppet resources. It is structured as a JSON object, conforming to the
+[report wire format v6][reportv6].
+
+## Deprecated Commands
+
 ### "store report", version 5
 
-The version 5 store report command differs from version four in the addition of
-a "noop" flag, which is a boolean indicating whether the report was produced by
-a puppet run with --noop, as well as in the conversion of dash-separated fields to
+> **Note:** This version is deprecated, use the latest version instead.
```

Comment:
I'd drop this -- it's already under a deprecated header and we don't want to repeat the note for each deprecation.